src/app/templates/ts/tslint.json

```diff
@@ -0,0 +1,32 @@
+{
+  "extends": [
+    "tslint:recommended"
```

Comment:
Since we are shipping with Prettier support, thoughts on extending [tslint-config-prettier](https://github.com/alexjoverm/tslint-config-prettier)?